deploy/terraform/terraform-units/modules/sap_library/infrastructure.tf

```diff
@@ -188,30 +188,3 @@ resource "azurerm_private_dns_zone_virtual_network_link" "vault_additional" {
   tags                                 = var.infrastructure.tags
 }
 
-
```

Comment:
This was here for the scenario where you don't have a deployer but want to add an extra network

Reply:
It was a duplicate of the resource on line 85